mmdeploy/codebase/mmagic/deploy/super_resolution_model.py

```diff
@@ -194,25 +197,27 @@ def forward(self,
         Returns:
             list | dict: High resolution image or a evaluation results.
         """
+
         if hasattr(self.data_preprocessor, 'destructor'):
             inputs = self.data_preprocessor.destructor(
                 inputs.to(self.data_preprocessor.input_std.device))
+
         outputs = []
         for i in range(inputs.shape[0]):
             output = self.wrapper.invoke(inputs[i].permute(
-                1, 2, 0).contiguous().detach().cpu().numpy())
+                1, 2, 0).contiguous().detach().cpu().numpy() * 255.)
```

Comment:
why do we need to multiply by 255? Is there any corresponding code in mmagic?

Reply:
The input should be in range 0-255, or the model prediction is wrong.

Reply:
but we have normalization in pipeline.json, this should do the work. Could double check here?

` ``
                   {
                        "type": "Normalize",
                        "mean": [
                            0.0,
                            0.0,
                            0.0
                        ],
                        "std": [
                            255.0,
                            255.0,
                            255.0
                        ],
                        "to_rgb": false
                    },
```